dataland-frontend/tests/e2e/specs/company-metadata/CompaniesOnlySearch.ts

```diff
@@ -25,3 +27,49 @@ describe("As a user, I want to be able to search companies existing on Dataland"
     verifyCompanySearchResultTable();
   });
 });
+
+describeIf(
+  "As a user, I want to be able to search companies by their alternative names",
+  {
+    executionEnvironments: ["developmentLocal", "development"],
+    dataEnvironments: ["fakeFixtures"],
+  },
+  function () {
+    beforeEach(() => {
+      cy.ensureLoggedIn();
+    });
+
+    let fixtureData: Array<{
+      companyInformation: CompanyInformation;
+      t: EuTaxonomyDataForFinancials;
+    }>;
+
+    before(function () {
+      cy.fixture("CompanyInformationWithEuTaxonomyDataForFinancials").then(function (companies) {
+        fixtureData = companies;
+      });
+    });
+
+    function getCompanyWithAlternativeName() {
+      return fixtureData.filter((it) => {
+        return (
+          it.companyInformation.companyAlternativeNames !== undefined &&
+          it.companyInformation.companyAlternativeNames.length > 0
+        );
+      })[0];
+    }
+
+    it("Search for company by its alternative name", () => {
+      const testCompany = getCompanyWithAlternativeName();
+      const searchValue = testCompany.companyInformation.companyAlternativeNames!![0];
+      cy.visitAndCheckAppMount("/companies-only-search");
```

Comment:
Warum hier `/companies-only-search` anstelle von `/companies`. Aus meiner Sicht ist `/companies-only-search` 
irrelevant und fliegt vsl. sowieso irgendwann raus

Reply:
Da habe ich keine starke Meinung zu, aber es könnte auch sinnvoll sein, einen Test dieses API-Features  im dataland-e2etest Projekt zu realisieren.

Reply:
Das war tatsächlich Absicht um den companies test nicht noch weiter zu überfrachten. Ich denke nicht das diese Seite wegfallen wird, es sei denn die Productvision hat sich deutlich von dem entfernt was bisher bezüglich dieser "Telefonbuch"-Seite kommuniziert wurde. Es ist aktuell nur nicht besonders hoch in der Prio.